go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	github.com/hashicorp/go-syslog v1.0.0
 	github.com/hashicorp/hcl v1.0.0
 	github.com/hashicorp/logutils v1.0.0
-	github.com/hashicorp/nomad/api v0.0.0-20220707195938-75f4c2237b28
+	github.com/hashicorp/nomad/api v0.0.0-20220829153708-e1e5bb1dcefb
```

Comment:
Before we merge, let's update this again to the current beta version (at least).